UNR-1782: Move Multiserver Shooter Tutorial onto Example Project (#1274)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

<%(Callout type="warn" message="This tutorial is available as a preview to demonstrate early multiserver functionality in the GDK. Please note we do not yet recommend development of multiserver games. For more information, please follow our [development roadmap](https://github.com/spatialos/UnrealGDK/projects/1) and [Unreal features support]({{urlRoot}}/unreal-features-support) pages.")%>	

**What the tutorial covers**<br/>

> In this tutorial you’ll implement cross-server remote procedure calls (RPCs) in a simple first person shooter example project. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#inspector) that players can seamlessly move between and shoot across. It will look something like this:

Reviewer: ```suggestion
>**This tutorial uses the Example Project from the GDK’s [setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)**.

In this tutorial you’ll implement cross-server remote procedure calls (RPCs) for simple first-person shooter functionality in the Example Project. The end result will be a multiplayer, cloud-hosted Unreal game running across multiple [server-workers]({{urlRoot}}/content/glossary#worker) that players can seamlessly move between and shoot across. It will look something like this:
```

This is how we introduce the multi deployment tutorial - https://docs.improbable.io/unreal/alpha/content/tutorials/deployment-manager/tutorial-deploymentmgr-intro

![]({{assetRoot}}assets/tutorial/cross-server-shooting.gif)

> The exercise demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal!

Reviewer: ```suggestion
The tutorial demonstrates that the workflows and iteration speed you’re used to as an Unreal developer are almost entirely unaltered by the GDK: it’s just like regular Unreal.
```

Have wanted to get rid of "exercise" and the exclamation marks for a long time.

<br/>------<br/>

> _2019-08-02 Page updated with limited editorial review_

Reviewer: ```suggestion
_2019-08-02 Page updated with limited editorial review: updated project name._</br>
_2019-04-30 Page updated with limited editorial review._
```

Author: doc out of date so will commit suggestions manually.
